### PR TITLE
ecosystem / pcf - rev 2

### PR DESCRIPTION
**Pull request by keithresar**

### CODE REVIEW

File: Ecosystem Partners/Marketplace Guides/getting-started-with-pivotal-cloud-foundry.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ http://pivotal.io/platform-as-a-service/pivotal-cloud-foundry
 
 Pivotal has integrated their Cloud Foundry technology with the CenturyLink Cloud platform.  The purpose of this KB article is to help the reader take advantage of this integration to achieve rapid time-to-value for this Cloud Foundry solution.
 
-<img src="/knowledge-base/images/pivotal_pcf/how_pcf_works.png" style="border:0;max-width:1140px;">
+<img src="/knowledge-base/images/pivotal_pcf/how_pcf_works.png" style="border:0;">
```
Comment (nathanyoung) [inline Ecosystem Partners/Marketplace Guides/getting-started-with-pivotal-cloud-foundry.md]: Can you change the image src to `../../images/pivotal_pcf/how_pcf_works.png` so that the images won't be broken in Github? The commit analyzer doesn't (yet) check img tags. I'll go through and fix existing broken img links.

You can see most of the images are broken: https://github.com/CenturyLinkCloud/PublicKB/blob/master/Ecosystem%20Partners/Marketplace%20Guides/getting-started-with-pivotal-cloud-foundry.md

Comment (keithresar) [inline Ecosystem Partners/Marketplace Guides/getting-started-with-pivotal-cloud-foundry.md]: Broken in GH but not in real life!

Last I did that there were translation problems.  Let me try again.

From: Nathan Young
Reply-To: CenturyLinkCloud/PublicKB
Date: Tuesday, August 4, 2015 at 11:22 AM
To: CenturyLinkCloud/PublicKB
Cc: Keith Resar
Subject: Re: [PublicKB] ecosystem / pcf - rev 2 (#264)


In Ecosystem Partners/Marketplace Guides/getting-started-with-pivotal-cloud-foundry.md<https://github.com/CenturyLinkCloud/PublicKB/pull/264#discussion_r36208665>:

> @@ -31,7 +31,7 @@ http://pivotal.io/platform-as-a-service/pivotal-cloud-foundry
>
>  Pivotal has integrated their Cloud Foundry technology with the CenturyLink Cloud platform.  The purpose of this KB article is to help the reader take advantage of this integration to achieve rapid time-to-value for this Cloud Foundry solution.
>
> -
> +


Can you change the image src to ../../images/pivotal_pcf/how_pcf_works.png so that the images won't be broken in Github? The commit analyzer doesn't (yet) check img tags. I'll go through and fix existing broken img links.

You can see most of the images are broken: https://github.com/CenturyLinkCloud/PublicKB/blob/master/Ecosystem%20Partners/Marketplace%20Guides/getting-started-with-pivotal-cloud-foundry.md

**Comment by keithresar** [comment]
Change implemented using two levels of indirection ../../.

Do we need 3 since this is a subcategory?

**Comment by nathanyoung** [comment]
nope, our import takes the /knowledge-base into account automatically.
